specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2020-10-01/storagecache.json

```
@@ -0,0 +1,2089 @@
+          "type": "string"
+        },
+        "smbServerName": {
+          "description": "The name (NetBIOS) used for the HPC Cache to join the Active Directory Domain. Length must not be greater than 15 and chars must be in list of [-0-9a-zA-Z_] char class.",
+          "pattern": "^[-0-9a-zA-Z_]{1,15}$",
+          "type": "string"
+        },
+        "domainJoined": {
+          "description": "This field indicates if the HPC Cache is joined to the Active Directory Domain.",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Yes",
+            "No",
+            "Error"
+          ],
+          "x-ms-enum": {
+            "name": "DomainJoinedType",
+            "modelAsString": true
+          }
+        },
+        "credentials": {
+          "description": "Active Directory admin or user credentials used to join the HPC Cache to a domain.",
+          "properties": {
+            "username": {
+              "description": "User name of the Active Directory domain administrator. This value is stored encrypted and not returned on response.",
+              "type": "string"
+            },
+            "password": {
+              "description": "Plain text password of the Active Directory domain administrator. This value is stored encrypted and not returned on response.",
+              "x-ms-secret": true,
+              "type": "string"
+            }
+          },
+          "required": [
+            "username",
+            "password"
+          ]
+        }
+      },
+      "type": "object",
+      "required": [
+        "primaryDnsIpAddress",
+        "secondaryDnsIpAddress",
+        "domainName",
+        "smbServerName"
+      ]
+    },
+    "CacheUsernameDownloadSettings": {
+      "description": "Settings for username and group download for Extended groups.",
+      "type": "object",
+      "properties": {
+        "extendedGroupsEnabled": {
+          "type": "boolean",
+          "description": "This indicates if Extended Groups is enabled."
+        },
+        "usernameSource": {
+          "description": "This setting determines how the system gets username and group names for clients.",
+          "enum": [
+            "AD",
+            "LDAP",
+            "File",
+            "None"
+          ],
+          "x-ms-enum": {
+            "name": "UsernameSource",
+            "modelAsString": true
+          },
+          "type": "string",
+          "default": "None"
+        },
+        "groupFileURI": {
+          "type": "string",
+          "description": "The URI of the file containing the group information (in etc/group file format).  This field must be populated when 'usernameSource' is set to 'File'."
+        },
+        "userFileURI": {
+          "type": "string",
+          "description": "The URI of the file containing the user information (in etc/passwd file format). This field must be populated when 'usernameSource' is set to 'File'."
```

```suggestion
          "description": "The URI of the file containing the user information (in /etc/passwd file format). This field must be populated when 'usernameSource' is set to 'File'."
```

Missed leading "/" in the path to /etc/*.

---

Fixed in next commit.